src/PolyfillTrait.php

```diff
@@ -14,14 +14,7 @@
 if (version_compare(\PHPUnit\Runner\Version::id(), '7.0.0') < 0) {
```

Comment:
should we have polyfill by PHPUnit target?
eg, one may want to enable polyfill of methods added in PHPUnit 9, but does not want to enable polyfill of methods added in PHPUnit 8 (Etc)

Reply:
As a medium-term goal, certainly yes.

Reply:
And, for example, there could be a polyfill for `expectExceptionMessageMatches` for PHPUnit <7 and for >=7